### ISSUE TITLE
`typeof` `Response` and `Blob` are mixed up

### ISSUE THREAD

**Issue by bogeychan** [oven-sh/bun#717]
### Version

0.1.4

### Platform

WSL2 Ubuntu 

### What steps will reproduce the bug?

Execute the following code snippet using `bun run index.ts`:

```ts
// index.ts
const response = new Response('');

console.log(`typeof Response class: ${typeof Response}`);
console.log(`typeof response object: ${typeof response}`);
```

### How often does it reproduce? Is there a required condition?

Reproduce each time

### What is the expected behavior?

Console output:

```bash
typeof Response class: function
typeof response object: object
```

### What do you see instead?

Console output:

```bash
typeof Response class: object
typeof response object: function
```

### Additional information

I also observed the same problem with `Blob`, maybe other classes are also affected if not all of them:
```ts
const blob = new Blob();

console.log(`typeof Blob class: ${typeof Blob}`);
console.log(`typeof blob object: ${typeof blob}`);
```

```bash
typeof Blob class: object
typeof blob object: function
```

`BigInt` as primitive wrapper object works as usual:
```ts
const foo = BigInt(42);

console.log(`typeof BigInt: ${typeof BigInt}`);
console.log(`typeof foo: ${typeof foo}`);
```

```bash
typeof BigInt: function
typeof foo: bigint
```

**Comment by Jarred-Sumner** [comment]
This is an interesting side effect of using JSC C API

**Comment by Electroid** [comment]
These are all now fixed, as of `0.2.2`.

```
typeof Response class: function
typeof response object: object
typeof Blob class: function
typeof blob object: object
typeof BigInt: function
typeof foo: bigint
```
